docs/DeveloperGuide.md

```diff
@@ -562,19 +582,135 @@ The following activity diagram summarizes what happens when a user executes the
 <div style="page-break-after: always;"></div>
 
 #### Design consideration
-##### Aspect: Definition of number of high risk location of infection
-When displaying the list of high risk location of infection, the top few locations that has been most visited by any 
-infected people are wanted. 
+##### Aspect: User input for number of high risk locations
+The parameter for list high risk locations command is optional. Users can choose to or choose not to input the number of
+high risk locations for this command. 
+
+**Rationale**
+Instead of always using the pre-defined rule within VirusTracker, this implementation allows users to customize the
+number of high risk locations displayed in VirusTracker. If the pre-defined rule is in used, users might not be able to 
```

Comment:
Small grammatical error. `... is in used` -> `is in use`.

Reply:
Fixed.